From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Michael Ikey Doherty <michael.i.doherty@intel.com>
Date: Wed, 9 Apr 2014 12:48:41 +0100
Subject: [PATCH] Explicitly set beecrypt include directory

Signed-off-by: Michael Ikey Doherty <michael.i.doherty@intel.com>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 0d1bf81..c2b8a03 100644
--- a/configure.ac
+++ b/configure.ac
@@ -269,6 +269,7 @@ if test "$with_beecrypt" = yes ; then
     ])
   fi
 fi
+WITH_BEECRYPT_INCLUDE="-I /usr/include/beecrypt"
 AC_SUBST(WITH_BEECRYPT_LIB)
 AC_SUBST(WITH_BEECRYPT_INCLUDE)
 
