From b11d4643827b498a6165fa2b18e95eaf013a3392 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Fri, 13 Jan 2017 16:40:17 +0000
Subject: [PATCH] build for python3

Signed-off-by: Thiago Macieira <thiago.macieira@intel.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index c2b8a032a..34ac3e83d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -579,7 +579,7 @@ AS_IF([test "$enable_python" = yes],[
       ])
     CPPFLAGS="$save_CPPFLAGS"
     save_LIBS="$LIBS"
-    AC_SEARCH_LIBS([Py_Main],[python${PYTHON_VERSION} python],[
+    AC_SEARCH_LIBS([Py_Main],[python${PYTHON_VERSION} python3 python3.7m python],[
       WITH_PYTHON_LIB="$ac_res"
     ],[AC_MSG_ERROR([missing python library])
     ])
-- 
2.18.0

