From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Tue, 11 Apr 2017 00:58:59 +0000
Subject: [PATCH] skip datasync

Signed-off-by: Thiago Macieira <thiago.macieira@intel.com>
---
 lib/backend/db3.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/backend/db3.c b/lib/backend/db3.c
index 01fce27..226f793 100644
--- a/lib/backend/db3.c
+++ b/lib/backend/db3.c
@@ -302,6 +302,8 @@ static int dbiSync(dbiIndex dbi, unsigned int flags)
     DB * db = dbi->dbi_db;
     int rc = 0;
 
+    return rc;
+
     if (db != NULL && !dbi->dbi_no_dbsync) {
 	rc = db->sync(db, flags);
 	rc = cvtdberr(dbi, "db->sync", rc, _debug);
