From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 7 Sep 2017 14:52:54 -0700
Subject: [PATCH] rpm: use 'localhost' as hostname for building all packages.

Signed-off-by: Thiago Macieira <thiago.macieira@intel.com>
---
 build/pack.c | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/build/pack.c b/build/pack.c
index d311d5d..27de3c5 100644
--- a/build/pack.c
+++ b/build/pack.c
@@ -176,16 +176,9 @@ static const char * buildHost(void)
 {
     static char hostname[1024];
     static int oneshot = 0;
-    struct hostent *hbn;
 
     if (! oneshot) {
-        (void) gethostname(hostname, sizeof(hostname));
-	hbn = gethostbyname(hostname);
-	if (hbn)
-	    strcpy(hostname, hbn->h_name);
-	else
-	    rpmlog(RPMLOG_WARNING,
-			_("Could not canonicalize hostname: %s\n"), hostname);
+	strncpy(hostname, "localhost", 10);
 	oneshot = 1;
     }
     return(hostname);
