From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Tue, 2 Oct 2018 11:26:29 -0700
Subject: [PATCH] Force locale files not to be executable.

---
 scripts/find-lang.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/scripts/find-lang.sh b/scripts/find-lang.sh
index 315d491..26472c3 100755
--- a/scripts/find-lang.sh
+++ b/scripts/find-lang.sh
@@ -97,13 +97,15 @@ while test $# -gt 0 ; do
     esac
 done    
 
+echo "%defattr(0644,root,root,0755)" > $MO_NAME
+
 find "$TOP_DIR" -type f -o -type l|sed '
 s:'"$TOP_DIR"'::
 '"$ALL_NAME$MO"'s:\(.*/locale/\)\([^/_]\+\)\(.*\.mo$\):%lang(\2) \1\2\3:
 '"$NO_ALL_NAME$MO"'s:\(.*/locale/\)\([^/_]\+\)\(.*/'"$NAME"'\.mo$\):%lang(\2) \1\2\3:
 s:^\([^%].*\)::
 s:%lang(C) ::
-/^$/d' > $MO_NAME
+/^$/d' >> $MO_NAME
 
 find "$TOP_DIR" -type d|sed '
 s:'"$TOP_DIR"'::
