--- a/configure.ac	2019-11-05 14:08:34.182735570 -0800
+++ b/configure.ac	2019-11-05 14:09:08.866067747 -0800
@@ -1003,8 +1003,8 @@
   LIBS="$LIBS -ldmalloc"
 fi
 
-user_with_uid0=$(awk -F: '$3==0 {print $1;exit}' /etc/passwd)
-group_with_gid0=$(awk -F: '$3==0 {print $1;exit}' /etc/group)
+user_with_uid0=$(awk -F: '$3==0 {print $1;exit}' /usr/share/defaults/etc/passwd)
+group_with_gid0=$(awk -F: '$3==0 {print $1;exit}' /usr/share/defaults/etc/group)
 AC_DEFINE_UNQUOTED([UID_0_USER],["$user_with_uid0"],[Get the user name having userid 0])
 AC_DEFINE_UNQUOTED([GID_0_GROUP],["$group_with_gid0"],[Get the group name having groupid 0])
 
